tests/unit/aiplatform/test_cloud_training_job.py

```diff
@@ -143,7 +143,7 @@ def test_create_vertex_model_cloud_class(self):
         )
 
         my_model = LinearRegression(2, 1)
-        my_model.training_mode = "cloud"
+        my_model.remote = True
```

Comment:
Tests should be added for the following:

1. Remote training to remote prediction
2. Remote training to local prediction
3. Local training to remote prediction
4. Jupyter source retrieval